app/controllers/memberships_controller.rb

```diff
@@ -76,20 +75,12 @@ def edit
         Rails.logger.info("Bid placed successfully")
         flash[:notice] = 'Dein Mitgliedsbeitrag wurde erfolgreich gespeichert'
         MemberMailer.with(name: update_params[:name], email: update_params[:email], bid: update_params[:amount].to_f, membership_id: update_params[:id]).notify_bid.deliver_later
+        redirect_to membership_path(params[:id])
       else
         Rails.logger.error("Failed to place bid: #{response.inspect}")
         flash[:error] = JSON.parse(response.body)
+        render :show
```

Comment:
Now by rendering `:show` there is no `@membership` variable anymore? A `redirect_to` could be an option too, like in the line 78? Then we fetch the member again and we avoid that the user sends the form again by trying to reload?

Reply:
That's a pretty good point. I wasn't sure about that point. I use `redirect_to` now also passing the flash message.